### ISSUE TITLE
Expedition only mode stuck at return screen

### ISSUE THREAD

**Issue by pakulu** [EchoControl/Gunkolle#8]
**Desktop:**
What graphics card are you using?
1050 ti
**Config File:**
[Do Not Modify]
Busy=0
[Variables]
SortieInterval=1000
MaxRandomWaitS=3000
MinRandomWaitS=3000
MinRandomWait=0
MaxRandomWait=0
WINID=ahk_class LDPlayerMainFrame
PauseMn=00
ResumeHr=12
ResumemN=00
Doll1=AR15
Doll2=M4SOPMODII
Doll3=ERROR
Doll4=ERROR
WeaponType=AssaultRifle
Enhancement=0
ProductionTdoll=0
ProductionEquipment=0
FriendCollector=0
BatteryCollector=0
AutoBattleResend=0
WorldSwitcher=
LastXS=814
LastYS=887
[GPU]
AMD=0
#For AMD or others GPU's change value 0 to 1. For NVidia value must be 0

#combatsims can be of values 1,2, or 3
CombatSimsData=2

LastXS=860
LastYS=775

**Describe the bug**
Some echelons are sent back into logistics, some are not. This causes one echelon to return and get stuck on the result screen and totally stopping all action
Bot shows exponlycheck 1, ill try to screenshot it asap

**Screenshots**
Add screenshots with your android player and where the script has stopped.

**Comment by EchoControl** [comment]
Is it when one logistics return gets over another?

**Comment by pakulu** [comment]
Now that you mention it, the return times are indeed very similar so that might be the cause
![image](https://user-images.githubusercontent.com/44754208/59978990-48522000-95e2-11e9-9f97-87577a0b77df.png)
I may not have explained it properly regardless since im not too used to GFLs slang, so ill try to simplify it to avoid confusing you any further;
Echelon 1 finishes logistic->Bot repeats quest-> Echelon 2 finishes logistic->Bot gets stuck

**Comment by EchoControl** [comment]
Is it getting stuck at resend buttons or at logi return screen?
UPD. And what time is between 1 and 2 logis

**Comment by pakulu** [comment]
At logistic return screen
Last time i saw it happen there was like a 1~minute difference, between each echelon arriving from logistics

**Comment by EchoControl** [comment]
Yeah, I saw this bug. I can suggest sending logis so there is 3 or 7 mins between each, so overlaying won't happen.

**Comment by pakulu** [comment]
I see, i thought it might have just been me doing something wrong.
Ill keep that in mind, thanks for your help

**Comment by EchoControl** [comment]
You're always welcome!

**Comment by pakulu** [comment]
So 17 minutes passed and logistics are over. Echelon 4 had the shortest timer and unfortunately stuck the bot as such : 
![image](https://user-images.githubusercontent.com/44754208/59979230-2908c200-95e5-11e9-89c1-fd427939ef6a.png)
I manually resent it after waiting a few seconds and then echelon 1 arrived, which was effectively sent back to work by the bot.
Could it be its chekcing for 1 echelon only rather than all 4? 
EDIT: Left, came back, skorpion's echelon got bugged again, manually clicked it, all other 3 went smoothly, only after clicking on ahk again.
Maybe thats what im doing wrong, having my emulator being the last program i interact with before leaving it to its business?

**Comment by EchoControl** [comment]
It's wierd, cuz script find that logi arrives by part of screen circled. Your screen seems to lack it, or maybe the screen shot isn't full.
![](https://user-images.githubusercontent.com/49132081/59982014-d1824a80-9614-11e9-90b6-10358c275bad.jpg)
 Anyway, I'll update the ref. pic so it won't happen again.

**Comment by pakulu** [comment]
Yeah it was poorly cropped by my side, im gonna make sure last program i interact with is ahk and see if it works properly that way

**Comment by EchoControl** [comment]
I just tried it with scorpion, seems to work, but just in case, I've updated [ExpeditionArrive](https://github.com/EchoControl/Gunkolle/commit/1ee78a273910ed77b63d2f7ff09be8dd0e901a97). Please update it from your side and say if it works or not.

**Comment by pakulu** [comment]
Sorry for the late reply, works like a charm now! 
I'd like to take this chance and not open a new issue, does the "[ ] corpse dragging off" work on this version? Seems to refill my HG's ammo on 4-3e. Also seems like the map wont drag even when creating a macro like the old wiki shows

**Comment by EchoControl** [comment]
Nope, it's disabled for time being
